src/main/scala/org/broadinstitute/hail/check/Gen.scala

```diff
@@ -34,8 +43,8 @@ object Gen {
     a
   }
 
-  def getPartition(parts: Int) : Gen[Array[Int]] = Gen { p => partition(p.rng, p.size, parts) }
-  def getSize : Gen[Int] = Gen { p => p.size }
+  def partition(parts: Int) : Gen[Array[Int]] = Gen { p => partition(p.rng, p.size, parts) }
+  def size : Gen[Int] = Gen { p => p.size }
```

Comment:
Yeah. I think I prefer `Gen.size` since it seems the more primitive of the two, but if `sized` is frequently used it's probably worth the short-hand.